packages/asc-ui/src/components/Image/ImageStyle.ts

```diff
@@ -1,5 +1,17 @@
-import styled from '@datapunt/asc-core'
+import styled, { css } from '@datapunt/asc-core'
 
-export default styled.img`
+export type Props = {
+  square?: boolean
+} & React.ImgHTMLAttributes<HTMLImageElement>
```

Comment:
Please move the React.Img part to the react file, as we want to keep our *Style.ts free from react :)

Reply:
Moved them to `Image.tsx`